### PR TITLE
Changing the default base date of the Tree View

### PR DESCRIPTION
**Pull request by kerzhner**
This PR intends to solve the following pain points:
1. We have a daily DAG with a non-midnight start date. When we click on the Tree View icon on http://myairflowinstance.com/admin/, rightmost DAG instance in the Tree View is not the latest instance of the DAG. It is the second to latest instance.
2. For other DAGs with midnight start dates, the Tree View rightmost DAG instance is often a fully finished DAG instance. It is nice to see at least one column of an un-started DAG instance. This makes it clear what the next DAG instance is and when it will run.

In other words, we currently see this:
screen shot 2015-09-23 at 2 22 51 pm
But this is clearer:
screen shot 2015-09-23 at 2 30 51 pm

### CODE REVIEW

File: airflow/www/app.py [hunk 1]
```diff
@@ -1055,7 +1055,7 @@ def tree(self):
         num_runs = int(num_runs) if num_runs else 25
 
         if not base_date:
-            base_date = dag.latest_execution_date or datetime.now()
+            base_date = (dag.latest_execution_date + 2 * dag.schedule_interval) or datetime.now()
```
Comment (mistercrunch) [inline airflow/www/app.py]: In the case of a new DAG dag.latest_execution_date will be `None` and this will raise an error, you'll need an `if` here...

**Comment by kerzhner** [comment]
@mistercrunch Good catch. Fixed in https://github.com/kerzhner/airflow/commit/8f642753fb701801aad5d23ec2006697b3657335.
